src/App.js

```diff
@@ -32,45 +31,25 @@ function App() {
   const [showNavBar, setShowNavBar] = useState(false);
   const userAuth = useAuth();
 
-  onAuthStateChanged(auth, (user) => {
```

Comment:
This function got deleted, which could be the reason why the navbar is not working correctly

Reply:
using setShowNavBar to toggle whether the navbar is shown or not, because userAuth is initialised as null and doesnt refresh (not sure why it doesnt refresh so i created another state that is independent of firebase auth login